src/notification.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * @param {event} event notification event
+ */
+export function handleNotification(event) {
+	console.debug('stt_helper: handleNotification', event)
+	if (event.notification.app !== 'stt_helper' || event.action.type !== 'WEB') {
+		return
+	}
+	console.debug('stt_helper: handleNotification', event)
+	event.cancelAction = true
+	fetchTranscript(event.notification.objectId)
+}
+
+/**
+ * @param {string} transcriptId transcript id
+ * @return {Promise<void>}
+ */
+async function fetchTranscript(transcriptId) {
+	const { default: axios } = await import(/* webpackChunkName: "axios-lazy" */'@nextcloud/axios')
+	const { generateUrl } = await import(/* webpackChunkName: "router-lazy" */'@nextcloud/router')
+	const { showError } = await import(/* webpackChunkName: "dialogs-lazy" */'@nextcloud/dialogs')
```

Comment:
Should we add a preload tag here, to make the modal show up faster?

Reply:
but that would mean loading it every time on all the pages until the notification remains uncleared as opposed to loading it only when the notification action is clicked on one page.

Reply:
We could also do a prefetch, that would preload the chunks only when the browser is idle

Reply:
yeah the network is not that much, the parsing of large js files on every page for nothing seems a bit heavy for the cpu and on lazy load does not cost that much even, it is almost realtime.